pkgs/tools/misc/routino/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchsvn, perl, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "Routino-${version}";
+  version = "3.2";
+
+  src = fetchsvn {
+    url = "http://routino.org/svn/trunk/";
```

Comment:
Why not http://routino.org/download/routino-3.2.tgz ?

Reply:
You're right, that's more elegant.